pom.xml

```diff
@@ -378,6 +378,7 @@
             <configuration>
               <executable>${jvm.executable}</executable>
               <arguments>
+                <argument>-Des.path.home=.</argument>
```

Comment:
What about: `<argument>-Des.path.home=${project.build.directory}/eshome</argument>`

So it will be cleaned with `mvn clean`.